src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -5149,7 +5151,7 @@ else if (e1 instanceof boolean[] && e2 instanceof boolean[])
      * @since 1.5
      */
     @SideEffectFree
-    public static @MinLen(2) String toString(@PolyNull @PolyInterned Object @Nullable [] a) {
+    public static @MinLen(2) String toString(@PolyMustCall @PolyNull @PolyInterned Object @PolyMustCall @Nullable [] a) {
```

Comment:
This `@PolyMustCall` doesn't make sense to me, for two reasons:
1. `toString` itself shouldn't care what `MustCall` types its arguments have
2. even if, for some reason, we do want to annotate this argument to `toString`, `@PolyMustCall` on both the component type and on the array itself seems wrong: the array itself won't have a must-call obligation (at least, as the MustCall type system is used by the RLC), and even if it does I don't see a reason why we need to LUB its type with the MustCall types of the array components. Can you explain the reason for these annotations?